docs/content/installation/database-preparation.en-us.md

```diff
@@ -45,7 +45,7 @@ Note: All steps below requires that the database engine of your choice is instal
 
     ```sql
     SET old_passwords=0;
-    CREATE USER 'gitea' IDENTIFIED BY 'gitea';
+    CREATE USER 'gitea'@'localhost' IDENTIFIED BY 'gitea';
```

Comment:
```suggestion
    CREATE USER 'gitea'@'%' IDENTIFIED BY 'gitea';
```

More universally usable, in case database is on another host or being connected to by IP address.

Reply:
Supposedly newer versions MariaDB dropped support for "%" in the hostname, I don't have an instance I can test with.

Reply:
Hmm, how is one supposed to create a "all hosts" user in these versions then?

Reply:
Actually I think `localhost` will not work because [of this](https://mariadb.com/kb/en/create-user/#host-name-component):

> On Linux, the loopback interface (127.0.0.1) will not match 'localhost' as it is not considered a local connection: this means that only connections via UNIX-domain sockets will match 'localhost'.

Gitea's default example config is `database.HOST = 127.0.0.1:3306`, so I'm pretty sure it's inoperable.

Reply:
Oh dear, Thanks for looking into this. I guess 127 would be the best in this case

Reply:
Why not keep it at "all hosts"? That should work for IPv6 as well.

Reply:
I was thinking because this section specifically mentions local connections, but I'm not attached to keeping it that way and could change to explicitly all hosts since that is in theory what we have now.

Reply:
Both variants are fine, but I prefer the explicit `'%'`, which I'm pretty sure will also work on unix sockets.